io.github.francivaldo4334.Bitsafe_cli.yaml

```diff
@@ -0,0 +1,30 @@
+id: io.github.francivaldo4334.Bitsafe_cli
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: bitsafe.sh
+modules:
+  - python3-requirements.json
+  - name: app-install
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/package/bitsafe
+      - cp -r bitsafe/ /app/package/
+      - cp -r bitsafe_ic.png /app/package/bitsafe_ic.png
+      - install -D setup.py /app/package/setup.py
+      - pip3 install /app/package
+    sources:
+      - type: file
+        path: setup.py
+      - type: dir
+        path: bitsafe/
+        dest: bitsafe/
```

Comment:
this doesn't exist.

Reply:
the comment was not to put it back.

Flatpak should get the source from the upstream repository. We do not host upstream projects. Just the the manifest to build the flatpak.